src/functions/types.ts

```diff
@@ -82,7 +82,8 @@ type FunctionInputRuntimeType<
       | typeof SlackSchemaTypes.user_id
       | typeof SlackSchemaTypes.usergroup_id
       | typeof SlackSchemaTypes.channel_id
-      | typeof SlackSchemaTypes.date ? string
+      | typeof SlackSchemaTypes.date
+      | typeof SlackSchemaTypes.rich_text ? string
```

Comment:
we should probably have this resolve to the `UnknownRuntimeType` (which is an `any`) - since it's a complex object at runtime and not a string. I believe you can just remove it from the checks here and it would fall into that condition below on line 88.

Reply:
ah are we saying to not include the entire line or to have the line `| typeof SlackSchemaTypes.rich_text ? UnknownRuntimeType`

Reply:
Good point, it would probably be better to be explicit here, so it would be more like adding a line at the end so we have another condition that matches specifically for rich text. So the block looks something like:

```ts
: Param["type"] extends
  | typeof SlackSchemaTypes.user_id
  | typeof SlackSchemaTypes.usergroup_id
  | typeof SlackSchemaTypes.channel_id
  | typeof SlackSchemaTypes.date ? string
: Param["type"] extends typeof SlackSchemaTypes.timestamp ? number
: Param["type"] extends typeof SlackSchemaTypes.rich_text ? UnknownRuntimeType
: UnknownRuntimeType;
```